rasa/core/training/story_conflict.py

```diff
@@ -304,17 +304,28 @@ def _get_previous_event(
     # A typical state is, for example,
     # `{'prev_action_listen': 1.0, 'intent_greet': 1.0, 'slot_cuisine_0': 1.0}`.
     # We need to look out for `prev_` and `intent_` prefixes in the labels.
```

Comment:
I guess the example is not correct anymore. Please update.

Reply:
Done